core/src/main/java/org/jacorb/poa/RequestProcessor.java

```diff
@@ -361,8 +380,11 @@ else if (servant instanceof org.omg.PortableServer.DynamicImplementation)
                                  " opname: " + operation +
                                  " invoke operation on servant (dsi based)");
                 }
-                if( specialOperations.contains(operation) &&
-                    !(servant instanceof org.jacorb.orb.Forwarder) )
+                SpecialOps special_op = specialOperations.getOrDefault(operation, SpecialOps.NONE);
```

Comment:
getOrDefault is JDK 8. Currently JacORB is using 1.6 (with a possibility of using 1.7)